### PR TITLE
Dashboard

### PR DESCRIPTION
**Pull request by dtjv**
- Add dashboard client route and UI to display decks
- Add mongodb connection
- Add server endpoint to retrieve decks

### CODE REVIEW

File: src/client/components/Dashboard.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import DeckItem from './DeckItem';
+// import _ from 'underscore';
```
Comment (mvargeson) [inline src/client/components/Dashboard.js]: Please remove commented code

File: src/client/components/Dashboard.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+import $ from 'jquery';
+
+// stub some data
```
Comment (mvargeson) [inline src/client/components/Dashboard.js]: If we are now using real data then please remove this commented code as well.

File: src/client/components/Dashboard.js [hunk 3]
```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import DeckItem from './DeckItem';
+// import _ from 'underscore';
+import $ from 'jquery';
```
Comment (mvargeson) [inline src/client/components/Dashboard.js:3]: Do we intend to use jQuery for anything but HTTP requests? If not then we may have an opportunity to refactor at some point using `fetch` with a polyfill for browsers other than Chrome and Mozilla.

File: src/server/db/index.js [hunk 4]
```diff
@@ -1,5 +1,5 @@
-// import mongoose from 'mongoose';
+import mongoose from 'mongoose';
 
-// export default {
-//   connect: () => mongoose.connect('mongodb://localhost:27017/wonky'),
-// };
+export default {
+  connect: () => mongoose.connect('mongodb://localhost:27017/wonky'),
```
Comment (mvargeson) [inline src/server/db/index.js]: Connection info should be in a configuration file.

File: src/server/routes/api.js [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+    Decks.find({}).then((decks) => {
+      console.log(decks);
+      res
+        .status(200)
+        .type('json')
+        .json(decks);
+    });
+  });
+
+// router.route('/api/decks/:deckId/card/:cardId')
```
Comment (mvargeson) [inline src/server/routes/api.js]: Commented code should be removed.

File: src/client/components/Splash.js [hunk 6]
```diff
@@ -7,7 +7,7 @@ const Splash = () => (
     <br />
     <h1 className="center"> WonkyFlash </h1>
```
Comment (mvargeson) [inline src/client/components/Splash.js:8]: The wonkiest

File: src/server/routes/api.js [hunk 7]
```diff
@@ -0,0 +1,24 @@
+import { Router } from 'express';
+import Decks from '../models/decks';
+
+const router = new Router();
+
+router.route('/api/decks')
+  .get((req, res) => {
+    Decks.find({}).then((decks) => {
+      console.log(decks);
```
Comment (mvargeson) [inline src/server/routes/api.js]: Please remove console logs

File: src/client/components/Dashboard.js (hunk 3 above)
Comment (dtjv) [inline src/client/components/Dashboard.js:3]: hmm... jQuery is a dependency for Materialize (if we use certain features). Use of jQuery here was to get something to work quickly. I agree, that it should be refactored at some point. I'll make a task for later to get this PR out.

Comment (mvargeson) [inline src/client/components/Dashboard.js:3]: Sounds good. I didn't realize it was a materialize dep. If so then I'm cool to use it here.

**Pull request merged**